IntegrationTests/PurchaseTester/PurchaseTester.xcodeproj/xcshareddata/xcschemes/WatchPurchaseTester - StoreKit Config Files (Xcode 12+).xcscheme

```diff
@@ -77,27 +75,16 @@
       savedToolIdentifier = ""
       useCustomWorkingDirectory = "NO"
       debugDocumentVersioning = "YES">
-      <RemoteRunnable
-         runnableDebuggingMode = "2"
-         BundleIdentifier = "com.apple.Carousel"
-         RemotePath = "/PurchaseTester">
+      <BuildableProductRunnable
+         runnableDebuggingMode = "0">
          <BuildableReference
             BuildableIdentifier = "primary"
             BlueprintIdentifier = "2D54BF602437DED800FF4EE4"
             BuildableName = "WatchPurchaseTester.app"
             BlueprintName = "WatchPurchaseTester"
             ReferencedContainer = "container:PurchaseTester.xcodeproj">
          </BuildableReference>
-      </RemoteRunnable>
-      <MacroExpansion>
-         <BuildableReference
-            BuildableIdentifier = "primary"
-            BlueprintIdentifier = "2D54BF602437DED800FF4EE4"
-            BuildableName = "WatchPurchaseTester.app"
-            BlueprintName = "WatchPurchaseTester"
-            ReferencedContainer = "container:PurchaseTester.xcodeproj">
-         </BuildableReference>
-      </MacroExpansion>
+      </BuildableProductRunnable>
    </ProfileAction>
    <AnalyzeAction
       buildConfiguration = "Debug">
```

Comment:
I guess all these changes come from Xcode 13 right?

Reply:
Yup, exactly. I'm sure this will end up changing again between now and Xcode 13 GM 🙃